connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfigTransformer.java

```diff
@@ -58,21 +72,19 @@ private void scheduleReload(String connectorName, Map<String, Long> ttls) {
     }
 
     private void scheduleReload(String connectorName, String path, long ttl) {
-        Herder herder = worker.herder();
-        if (herder.connectorConfigReloadAction(connectorName) == Herder.ConfigReloadAction.RESTART) {
```

Comment:
seems like we no longer use `connectorConfigReloadAction` anywhere but tests. Can we remove it from the herders and maybe clean up/remove/update some tests?